code/modules/mob/dead/new_player/login.dm

```diff
@@ -1,4 +1,6 @@
 /mob/dead/new_player/Login()
+	if(config.use_exp_tracking)
+		client.set_exp_from_db()

```

Comment:
Ok. with the new changes back to the old updating version, the exp is read from the db before each update. The only need to read without updating would be in the lobby for getting job unlock times